Lib/axisregistry/__init__.py

```diff
@@ -1,52 +1,458 @@
+from copy import deepcopy
+from fontTools.ttLib import TTFont
+from fontTools.otlLib.builder import buildStatTable
+from fontTools.varLib.instancer.names import _updateUniqueIdNameRecord, NameID
+from fontTools.ttLib.tables._f_v_a_r import NamedInstance
 from pkg_resources import resource_filename
 from google.protobuf import text_format
-
+from collections import OrderedDict
 from axisregistry.axes_pb2 import AxisProto
+from collections import defaultdict
+from itertools import chain
+import logging
+from glob import glob
+import os
 
 try:
     from ._version import version as __version__  # type: ignore
 except ImportError:
     __version__ = "0.0.0+unknown"
 
+log = logging.getLogger(__file__)
+
+# TODO we may have more of these. Please note that some applications may not
+# implement variable font style linking.
+LINKED_VALUES = {
+    "wght": {400.0: 700.0},
+    "ital": {0.0: 1.0},
+}
+
+# Static font styles. The GF api only support the following static font styles
+GF_STATIC_STYLES = OrderedDict(
```

Comment:
I feel like we have this everywhere. I recently used `gftools.util.google_fonts._KNOWN_WEIGHTS`

Reply:
Since this repo is going to be used in gftools, I plan to remove `gftools.util.google_fonts._KNOWN_WEIGHTS` and use this one instead.